docs/source/models/intervention_models/alzheimers/testing_diagnosis.rst

```diff
@@ -30,4 +30,222 @@
+
+
+Intervention Overview
+-----------------------
+
+Alzheimer's testing is classified into two groups - baseline (CSF and PET) testing, 
+and BBBM testing. Baseline testing already exists in present-day conditions, before the 
+hypothetical introduction of BBBM testing in :ref:`Alternative Scenario 1 <alz_scenarios>`.
+The introduction of BBBM testing will replace some number of the baseline tests in 
+Alternative Scenario 1, and will also be used to inform treatment in Alternative 
+Scenario 2. 
+
+Vivarium Modeling Strategy
+--------------------------
+
+Baseline testing - CSF and PET
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CSF and PET are existing biomarker tests which can identify cases of Alzheimer's disease.
+We model CSF and PET tests in our :ref:`Reference Scenario <alz_scenarios>` in order to
+estimate the number of these more expensive, invasive tests which would be replaced
+by the introduction of BBBM tests in :ref:`Alternative Scenario 1 <alz_scenarios>`.
+
+Location-specific testing rates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+US testing rates among patients who are diagnosed with MCI or dementia are given 
+in Figure 3 in [Yan-et-al-2025-Medicare-Diagnostics]_ (overall cohort).
+
+We adjust these US CSF and PET testing rates upwards (2x) due to subsequent 
+increases in coverage by Medicare, and add substantial uncertainty (-50% for the lower 
+bound and +50% for the upper bound) to create a 95% confidence interval.
+
+To reach CSF and PET testing rates for other countries, we adjust the (increased) US test rates based on 
+per capita Alzheimer's pharmaceutical spending from [Brain-Health-Atlas]_.
+For example, a country spending 10% of the US would have CSF and PET testing rates equal to 
+10% of the US.
+
+Due to a high perentage of CSF testing in Sweden [Mattke-et-al-2024-Sweden-Capacity]_, 
+we further adjust Sweden's CSF rate to be 9x its PET rate from the previous step.
+
+We use these location-specific test rates to determine how likely a simulant with 
+MCI is to receieve a CSF or PET test, and which test they should receive.
+
+.. list-table:: Location-specific test rates
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Location
+    - CSF mean % (Confidence)
+    - PET mean % (Confidence)
+  * - US
+    - 4.4 (2.2, 6.6)
+    - 1.4 (.70, 2.1)
+  * - Germany
+    - 2.8 (1.4, 4.2)
+    - .89 (.44, 1.3)
+  * - Spain
+    - 1.4 (.70, 2.1)
+    - .45 (.22, .67)
+  * - Sweden
+    - 3.8 (1.9, 5.7)
+    - .42 (.21, .63)
+  * - UK
+    - 1.3 (.65, 2.0)
+    - .41 (.21, .62)
+  * - Japan
+    - 1.3 (.65, 1.9)
+    - .41 (.21, .62)
+  * - Israel
+    - .43 (.21, .64)
+    - .14 (.07, .20)
+  * - Taiwan
+    - .42 (.21, .63)
+    - .13 (.07, .20)
+  * - Brazil
+    - .12 (.06, .18)
+    - .04 (.02, .06)
+  * - China
+    - .08 (.04, .12)
+    - .02 (.01, .04)
+
+Implementation
+^^^^^^^^^^^^^^
+Each simulant will be assigned a propensity (0/high to 1/low) for receiving a test. The propensity will apply for the 
+simulant's lifetime.
```

Comment:
Do we want to keep the same propensity throughout the simulant's lifetime, or do we want to update it on each timestep? If we don't update every timestep, then each simulant will either get a test on the first timestep they are eligible, or they'll never get a test (here I'm using the word "eligible" in the clinical sense, not taking the propensity into account as in the definition of eligibility in the docs). Does that seem right? Is that what we want?

I guess if we _did_ update the propensity every timestep, would that throw off our calibration? (I.e., would we get more testing than we want because each simulant has multiple chances to get tested?)

Reply:
This seems like something where it might be better to [do something in-between](https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_census_synthdata/concept_model.html#propensities), if you can figure out how to calibrate it

Reply:
I suggest we keep it simple, the exact timing is not particularly important.  We can note this as a potential area for improvement in the future.